### PR TITLE
Fix scripted diff check condition

### PR DESCRIPTION
**Pull request by UdjinM6**
Fixes the issue discovered in #3122

### CODE REVIEW

**Comment by UdjinM6** [comment]
Took me a while to figure it out but it finally works now _I think_.

There are 2 debug scripted-diff-s here:
3f8b89c - https://travis-ci.org/dashpay/dash/jobs/591689130#L658 (passed)
0b10151 - https://travis-ci.org/dashpay/dash/jobs/591689130#L673 (failed)
i.e worked as expected :)

File: test/functional/llmq-chainlocks.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def wait_for_chainlock(self, node, block_hash, timeout=15):
                 # block might not be on the node yet
                 pass
             time.sleep(0.1)
-        raise AssertionError("wait_for_chainlock timed out")
+        raise AssertionError("wait_for_chainlocked_block timed out")
```
Comment (PastaPastaPasta) [inline test/functional/llmq-chainlocks.py]: this diff should not be in this

File: test/functional/llmq-is-cl-conflicts.py [hunk 2]
```diff
@@ -228,7 +228,7 @@ def wait_for_chainlock(self, node, block_hash):
                 # block might not be on the node yet
                 pass
             time.sleep(0.1)
-        raise AssertionError("wait_for_chainlock timed out")
+        raise AssertionError("wait_for_chainlocked_block timed out")
```
Comment (PastaPastaPasta) [inline test/functional/llmq-is-cl-conflicts.py]: same

File: ci/build_src.sh [hunk 3]
```diff
@@ -12,7 +12,8 @@ unset DISPLAY
 export CCACHE_COMPRESS=${CCACHE_COMPRESS:-1}
 export CCACHE_SIZE=${CCACHE_SIZE:-400M}
 
-if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then contrib/devtools/commit-script-check.sh $TRAVIS_COMMIT_RANGE; fi
+if [ "$PULL_REQUEST" != "false" ]; then contrib/devtools/commit-script-check.sh $COMMIT_RANGE; fi
+
```
Comment (PastaPastaPasta) [inline ci/build_src.sh]: drop this line

**Comment by UdjinM6** [comment]
@PastaPastaPasta these changes are from 2 debug commits I included to show that patch actually works as expected, I'll drop them before merging.

**Comment by PastaPastaPasta** [comment]
Yup, once those are removed, I'm utACKed for merge

**Comment by UdjinM6** [comment]
Cool :) I'll just wait for more people to comment here before removal to make sure they had a change to review current results.

**Comment by UdjinM6** [comment]
Dropped 2 debug commits.

**Pull request merged**